src/main/java/org/broadinstitute/hellbender/tools/sv/cluster/CanonicalSVLinkage.java

```diff
@@ -173,30 +184,31 @@ private static boolean clusterTogetherWithParams(final SVCallRecord a, final SVC
         }
         final SimpleInterval intervalB1 = b.getPositionAInterval();
         final SimpleInterval intervalB2 = b.getPositionBInterval();
-        if (!(intervalA1.overlaps(intervalB1) && intervalA2.overlaps(intervalB2))) {
-            return false;
+        final boolean hasBreakendProximity = intervalA1.overlaps(intervalB1) && intervalA2.overlaps(intervalB2);
+        // Use short-circuiting statements since sample overlap is the least scalable / slowest check
+        if (hasReciprocalOverlapAndSizeSimilarity == null) {
```

Comment:
Here I used `null` as a distinct case where the variants are interchromosomal, in which case the logic is different from `False` as with intrachromosomal variants. I found the code to be clearest this way (it's a bit complicated since there are 3 booleans) although maybe there is a better way to write it?

Reply:
I see. I guess I'd advocating for just re-checking the intrachromosal status explicitly here again here instead of using null, but it's not a big deal either way.